--- openchange-openchange-2.3-VULCAN/configure.ac.omv~	2016-10-13 16:15:07.156403976 +0200
+++ openchange-openchange-2.3-VULCAN/configure.ac	2016-10-13 16:15:10.984348810 +0200
@@ -292,7 +292,7 @@ OC_RULE_ADD(testsuite, TESTSUITE)
 dnl ---------------------------------------------------------------------------
 dnl Check for nanomsg library (server notification)
 dnl ---------------------------------------------------------------------------
-PKG_CHECK_MODULES(NANOMSG, libnanomsg, [], [AC_MSG_ERROR(Install nanomsg library >= 0.5)])
+PKG_CHECK_MODULES(NANOMSG, nanomsg, [], [AC_MSG_ERROR(Install nanomsg library >= 0.5)])
 AC_SUBST(NANOMSG_CFLAGS)
 AC_SUBST(NANO_LIBS)
 
