--- openchange-2.2-NANOPROBE/Makefile.omv~	2015-02-20 23:53:04.265877136 +0100
+++ openchange-2.2-NANOPROBE/Makefile	2015-02-20 23:56:08.543527858 +0100
@@ -317,7 +317,7 @@ libmapi.$(SHLIBEXT).$(PACKAGE_VERSION):
 	libmapi/socket/interface.po			\
 	libmapi/socket/netif.po				
 	@echo "Linking $@"
-	@$(CC) $(DSOOPT) $(CFLAGS) $(LDFLAGS) -Wl,-soname,libmapi.$(SHLIBEXT).$(LIBMAPI_SO_VERSION) -o $@ $^ $(LIBS)
+	@$(CC) $(DSOOPT) $(CFLAGS) $(LDFLAGS) -Wl,-soname,libmapi.$(SHLIBEXT).$(LIBMAPI_SO_VERSION) -o $@ $^ $(LIBS) -L@LIBDIR@/samba -Wl,-rpath,@LIBDIR@/samba -lndr-samba
 
 
 libmapi.$(SHLIBEXT).$(LIBMAPI_SO_VERSION): libmapi.$(SHLIBEXT).$(PACKAGE_VERSION)
@@ -579,7 +579,7 @@ libmapiadmin.$(SHLIBEXT).$(PACKAGE_VERSI
 	libmapiadmin/mapiadmin.po 		\
 	libmapi.$(SHLIBEXT).$(PACKAGE_VERSION)
 	@echo "Linking $@"
-	@$(CC) $(DSOOPT) $(LDFLAGS) -Wl,-soname,libmapiadmin.$(SHLIBEXT).$(LIBMAPIADMIN_SO_VERSION) -o $@ $^ $(LIBS) $(LIBMAPIADMIN_LIBS) 
+	@$(CC) $(DSOOPT) $(LDFLAGS) -Wl,-soname,libmapiadmin.$(SHLIBEXT).$(LIBMAPIADMIN_SO_VERSION) -o $@ $^ $(LIBS) $(LIBMAPIADMIN_LIBS) -L@LIBDIR@/samba -Wl,-rpath,@LIBDIR@/samba -ldcerpc-samba
 
 
 
